graph/project-details/src/lib/ui/sourcemap-info-component.tsx

```diff
@@ -0,0 +1,30 @@
+interface SourceMapInfoProps {
+  sourceMap: Record<string, string[]>;
+  targetName: string;
+  key: string;
```

Comment:
It might be _ok_ but `key` in react is usually a reserved keyword, used to identify components when rendered via a looping function such as

```jsx
{ items.map(i => <MyItem item={i} key={uniqueId} />) }
```